thunder/core/jit_ext.py

```diff
@@ -915,6 +915,23 @@ def _general_jit_torch_autograd_function_apply_lookaside(obj: Any, *args, **kwar
     return _interpret_call(custom_forward, wrapped_ctx, *args_, **kwargs_)
 
 
+@run_once
+def _warn_custom_torch_info_function():
+    warnings.warn("`torch.finfo` is not implemented in Thunder. Calling `torch.finfo` directly.")
+
+
+@general_jit_lookaside(torch.finfo)
+def _general_jit_torch_finfo_lookaside(obj: Any):
+    _warn_custom_torch_info_function()
+    thunder_dtype = unwrap(obj)
+    torch_dtype = thunder.dtypes.to_torch_dtype(thunder_dtype)
+    res = torch.finfo(torch_dtype)
+
+    pr = ProvenanceRecord(PseudoInst.OPAQUE, inputs=[wrap_const(res).provenance])
```

Comment:
If we want to wrap this manually, I think the inputs should be `[wrap_const(torch.finfo).provenance, obj.provenance]` (so the things you need to know from the call).

Reply:
Oh, I did not fully understand how that worked. Thanks for the explanation!